icons/jetpackcompose/jetpackcompose-line-wordmark.svg

```diff
@@ -0,0 +1 @@
.2.365-.29.911-.47 1.6-.47zm6.18 0c.913 0 1.513.306 1.924.873.37.51.525 1.331.562 2.287H106.3l-.25.256c.013.684.195 1.28.554 1.715.36.435.899.69 1.543.69.822 0 1.434-.489 1.766-1.26h.479c-.323 1.039-1.07 1.683-2.28 1.683-.822 0-1.455-.287-1.9-.807-.446-.52-.705-1.291-.705-2.279 0-.986.268-1.777.722-2.316.455-.54 1.095-.842 1.907-.842zm-81.924.426a1.93 1.93 0 0 0-1.43.601c-.359.383-.574.92-.619 1.549l.248.27h3.541l.25-.25c0-1.19-.822-2.17-1.99-2.17zm46.703 0c-.628 0-1.167.277-1.52.757-.352.48-.529 1.146-.529 1.94 0 .8.176 1.464.53 1.943.353.48.892.754 1.519.754.623 0 1.162-.277 1.516-.752.354-.475.535-1.133.535-1.91 0-.816-.174-1.488-.526-1.973-.351-.484-.893-.76-1.525-.76Zm22.861 0c-.628 0-1.167.277-1.52.757-.352.48-.53 1.146-.53 1.94 0 .8.178 1.464.53 1.943.354.48.893.754 1.52.754a1.85 1.85 0 0 0 1.514-.752c.354-.475.535-1.133.535-1.91 0-.816-.172-1.488-.523-1.973-.352-.484-.893-.76-1.526-.76Zm12.348 0a1.93 1.93 0 0 0-1.43.601c-.359.383-.573.92-.619 1.549l.248.27h3.541l.25-.25c0-1.19-.822-2.17-1.99-2.17zm-71.903.01c-.604 0-1.129.288-1.475.768-.346.48-.527 1.14-.527 1.918 0 .777.181 1.438.527 1.918.346.48.871.767 1.475.767.62 0 1.16-.287 1.521-.763.36-.476.553-1.13.553-1.887 0-.79-.188-1.461-.547-1.947a1.867 1.867 0 0 0-1.527-.774Zm53.125 0c-.604 0-1.129.288-1.475.768-.346.48-.527 1.14-.527 1.918 0 .777.181 1.438.527 1.918.346.48.871.767 1.475.767.62 0 1.159-.287 1.52-.763.36-.476.554-1.13.554-1.887 0-.79-.19-1.461-.549-1.947a1.864 1.864 0 0 0-1.525-.774Zm-63.133.489c.803 0 1.296.605 1.404 1.42h-2.85c.077-.382.171-.753.381-.977.268-.286.623-.443 1.065-.443zm46.703 0c.496 0 .855.185 1.121.552.267.368.43.936.43 1.68 0 .699-.166 1.25-.436 1.611-.27.362-.634.551-1.115.551-.489 0-.85-.187-1.117-.55-.268-.364-.432-.923-.432-1.647 0-.718.165-1.277.434-1.643.268-.365.627-.554 1.115-.554zm22.861 0c.496 0 .855.185 1.121.552.267.368.428.936.428 1.68 0 .699-.166 1.25-.435 1.611-.27.362-.632.551-1.114.551-.489 0-.849-.187-1.117-.55-.268-.364-.433-.923-.433-1.647 0-.718.165-1.277.433-1.643.269-.365.63-.554 1.117-.554zm12.348 0c.803 0 1.296.605 1.404 1.42h-2.85c.077-.382.171-.753.382-.977.267-.286.623-.443 1.064-.443zM36.22 9.86c.478 0 .848.198 1.125.573.277.374.449.938.449 1.648 0 .671-.173 1.219-.451 1.586a1.347 1.347 0 0 1-1.123.564c-.453 0-.803-.191-1.069-.56-.266-.37-.433-.926-.433-1.625s.167-1.256.433-1.625c.266-.37.616-.56 1.069-.56zm53.125 0c.478 0 .848.198 1.125.573.277.374.449.938.449 1.648 0 .671-.175 1.219-.453 1.586s-.65.564-1.121.564c-.453 0-.805-.191-1.07-.56-.267-.37-.432-.926-.432-1.625s.165-1.256.431-1.625c.266-.37.618-.56 1.07-.56zm-45.477 1.946c-.43.197-1.206.24-1.892.392-.344.076-.673.18-.936.389a1.183 1.183 0 0 0-.437.947c0 .378.146.716.416.936.27.22.64.322 1.07.322 1.297 0 2.133-.838 2.133-1.63v-1.128zm-.146.53v.825c0 .407-.555 1.131-1.633 1.131-.35 0-.6-.084-.754-.209-.153-.125-.232-.29-.232-.549 0-.281.085-.425.248-.555.162-.129.423-.224.732-.292.508-.113 1.106-.192 1.639-.35z" transform="translate(-5.846 -5.611) scale(1.09134)"/></svg>
```

Comment:
This still has two paths. It needs to have only one. You can take a look at this comment to learn how to unite paths in [inkscape](https://inkscape.org/): https://github.com/devicons/devicon/issues/1306#issuecomment-1296014473

Reply:
> This still has two paths. It needs to have only one. You can take a look at this comment to learn how to unite paths in [inkscape](https://inkscape.org/): [#1306 (comment)](https://github.com/devicons/devicon/issues/1306#issuecomment-1296014473)

I've fixed it by uniting the paths in Inkscape as per the instructions. You can review the changes in commit [be4c378](https://github.com/devicons/devicon/pull/1839/commits/be4c37805812cb0e7a714b243cc4aef076747768).

---

> Only one issue left as far as I can see now :)
> 
> Also a preference thing, but I feel like the entire plain icon should be filled so that it looks something like this rough sketch: 
![jetpackcompose-plain-example](https://user-images.githubusercontent.com/43886029/277130222-35a503e6-ef53-499d-8d87-564946391e79.png)

I've made the changes according to your preference. Please review the commit [8c67d84](https://github.com/devicons/devicon/pull/1839/commits/8c67d84cf6faf9a9db002ae77e9132babd4139c7), and let me know if any further adjustments are needed.

Thanks.